src/world/area_mac/mac_00/7EAD10.c

```diff
@@ -5,6 +5,7 @@ static char* N(exit_str_1) = "mac_01";
 static char* N(exit_str_2) = "tik_19";
 static char* N(exit_str_3) = "kmr_20";
 static char* N(exit_str_4) = "mac_00";
+static char* N(exit_str_5) = "\0\0\0\0";
```

Comment:
Why is this required?

Reply:
because of some strange rodata padding issue. we'll probably figure it out when we actually do these files